config/locales/en.yml

```diff
@@ -564,3 +551,74 @@ en:
    new_ally_request:
      subject: "if me | %{ally_name} sent an ally request!"
      message: "<p>Please <a href=\"%{allies_url}\">sign in</a> to accept or reject the request!</p>"
+
+ devise:
+  password: "Password"
+  new_password: "New password"
+  confirmations:
+    resend_confirmation: "Resend confirmation instructions"
+  mailer:
+    confirmation:
+      link_instructions: "You can confirm your account email through the link below"
+      link_text: "Confirm my account"
+    reset_password:
+      link_instructions: "Someone has requested a link to change your password. You can do this through the link below."
+      link_text: "Change my password"
+      ignore: "If you didn't request this, please ignore this email."
+      info: "Your password won't change until you access the link above and create a new one."
+    unlock:
+      info: "Your account has been locked due to an excessive number of unsuccessful sign in attempts."
+      link_instructions: "Click the link below to unlock your account"
+      link_text: "Unlock my account"
+  passwords:
+    edit:
+      confirmation: "Confirm new password"
+      submit: "Change my password"
+    new:
+      send_instructions: "Send me reset password instructions"
+
+  registrations:
+    placeholders:
+      repeat_password: "Repeat Password"
+    edit:
+      labels:
+        profile_picture: "Profile picture"
+        remove_picture: "Remove picture"
+        password_for: "Password for <strong><%= t('app_name') %></strong> account"
+        update: "Update"
+      placeholders:
+        optional: "Optional"
+        about: "Introduce yourself! Why are you here and what are you looking to achieve? Please note this will be seen when users search for you"
+        required: "Required"
+      delete:
+        text: "Is <strong><%= t('app_name') %></strong> not working out for you?"
+        button: "Permanently delete my account"
+        confirm: "Are you sure? You'll lose all of your data. But you can always sign up again!"
+    new:
+      subtitle: "Start telling your stories and maintaining your health in a safe, private space"
+      label: "A few details"
+      sign_up: "Sign up"
+  sessions:
+    subtitle: "Welcome back, we're looking forward to you helping you tackle your day"
+    sign_in: 'Sign in'
+  shared:
+    sign_in_google: 'Sign in with Google'
+    sign_up_google: 'Sign up with Google'
+    password: 'Forgot your password?'
+    confirmation: "Didn't receive confirmation instructions?"
+    unlock: "Didn't receive unlock instructions?"
+  unlock:
+    resend: 'Resend unlock instructions'
+
```

Comment:
can we get rid of all of the extra whitespace here? just one newline should suffice :)